### DISCUSSION TITLE
label is ignored

### DISCUSSION THREAD

**Discussion by rabol** [filamentphp/filament#501] [Help]
**Describe the bug**
When using a component directly in a blade file the ->label() is ignored.

**To reproduce**
1. create a custom page
2. add this in your blade: {{ \Filament\Forms\Components\TextInput::make('test_input')->label('My label')->render() }}
3. the label will be 'Test Input' and not 'My Label'

**Expected behavior**
That the label for the input would be what is in the ->label() method

**Screenshots**

**Context**
- Package version: [e.g. 0.1.0]
- Livewire version: [e.g. 2.0.0]
- Laravel version: [e.g. 8.0.0]
- Browser: [e.g. Chrome, Safari]
- Server OS: [e.g. Ubuntu 18.04.4]
- PHP version: [e.g. 7.4]
- Database: [e.g. MySQL 8.0]

**Additional details**
Add any other details about the problem here.

**Comment by danharrin** [comment]
Yeah you should never use a v1 component on its own like that, what are you trying to do?

**Reply by rabol** [reply]
I'm juts trying to build a custom form where I control the layout
